src/main/java/mate/academy/rickandmorty/dto/internal/CartoonCharacterDto.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.rickandmorty.dto.internal;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Data;
+import mate.academy.rickandmorty.dto.external.LocationInputDataDto;
+
+@Data
+public class CartoonCharacterDto {
+    private Long id;
+    private Long externalId;
+    private String name;
+    private String status;
+    private String species;
+    private String type;
+    private String gender;
+    private LocationInputDataDto origin;
+    private LocationInputDataDto location;
+    private List<String> episode;
```

Comment:
The field 'episode' should be named 'episodes' to better reflect that it is a list of episodes.
```suggestion
    private List<String> episodes;
```